silk_ml/features.py

```diff
@@ -0,0 +1,60 @@
+import pandas as pd
+from scipy.stats import chi2_contingency, ttest_ind
+
+from .plots import plot_categorical, plot_numerical
+
+
+def split_classes(X, data, target, label):
+    ''' Returns the splited value of the dataset using the requested label
+
+    :param X: Main dataset with the variables
+    :type X: pd.DataFrame
+    :param data: Main dataset
+    :type data: pd.DataFrame
+    :param target: Categorical variable to classify
+    :type target: str or None
+    :param label: Name of the variable to split
+    :type label: str
+    :return: The `positive` and `negative` data splited
+    :rtype: tuple(pd.Series, pd.Series)
+    '''
+    positive = X.loc[data[target] == 1][label]
+    negative = X.loc[data[target] != 1][label]
+    return positive, negative
+
+
+def features_metrics(X, Y, targetname, plot=None):
+    ''' Checks for each variable the probability of being splited
```

Comment:
No sé si estoy entendiendo bien. Pero creo que debería decir algo así como:

```suggestion
    ''' Determines the likelihood from each variable of splitting correctly the dataset
```